linkerd/access-log/src/tower.rs

```diff
@@ -0,0 +1,169 @@
+use futures::TryFuture;
+use pin_project::pin_project;
+use std::future::Future;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+use std::time::{Duration, Instant};
+use tracing::{field, span, Level, Span};
+
+/// A tower layer that associates a tokio-tracing Span with each request
```

Comment:
nit: i would probably say
```suggestion
/// A tower layer that associates a `tracing` `Span` with each request
```